src/middleware/logger.js

```diff
@@ -0,0 +1,19 @@
+import chalk from 'chalk';
+import getRequestParams from '../utils/utils.js';
+
+function getFormattedDate() {
+  const date = new Date();
+  return `${date.getFullYear()}-${
+    date.getMonth() + 1
+  }-${date.getDate()} ${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
+}
+
+export default function logger(req, res, next) {
+  // eslint-disable-next-line no-console
+  console.log(
```

Comment:
please use your logger everywhere across project. In this case it will be easy to change logger library (from winston to smth else) without any impact for the project -> all you'll need to change is only logger.js module

Reply:
Done